renovate.json

```diff
@@ -1,11 +1,6 @@
 {
-	"extends": [ "config:base", "default:pinDigestsDisabled", "default:preserveSemverRanges" ],
```

Comment:
This was equivalent to `"rangeStrategy": "replace"`
(doc: https://docs.renovatebot.com/presets-default/#preservesemverranges)